mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.tmpl

```diff
@@ -199,6 +199,9 @@ func expandScheduling(v interface{}) (*compute.Scheduling, error) {
 		scheduling.LocalSsdRecoveryTimeout = transformedLocalSsdRecoveryTimeout
 		scheduling.ForceSendFields = append(scheduling.ForceSendFields, "LocalSsdRecoveryTimeout")
 	}
+	if v, ok := original["termination_time"]; ok {
+		scheduling.TerminationTime = v.(string)
```

Comment:
Continuing the discussion from https://github.com/GoogleCloudPlatform/magic-modules/pull/12791#discussion_r1977689485 - it sounds like ForceSendFields doesn't fix the problem of this not getting properly unset? It would definitely match the behavior of other string fields in this expand function, so they might also not behave as expected if this doesn't. (But that's outside the scope of this PR.)

I believe the fix would be to make sure that TerminationTime gets set to `""` (and added to ForceSendFields) if it's not present in `original`.

Reply:
I added workaround for this issue. Now everything is working as excepted.

Reply:
What was the workaround?

Reply:
I added force new to this field. Currently it's not possible to delete this field without destroying the old VM due to the API bug. Once there is no force new and `termination_time` is added to the forcesend, during deletion API is always trying to parse an empty string as a timestamp. One more improvement that could be added is creation of the customdiff that would set force new only when this field is deleted.
BTW other scheduling fields are also affected by this forcesend and could not be deleted once other scheduling fields still remain (I tested this for instance_termination_action).